OptimizelySDK/Config/PollingProjectConfigManager.cs

```diff
@@ -145,7 +147,7 @@ public bool SetConfig(ProjectConfig projectConfig)
                 return false;
             
             CurrentProjectConfig = projectConfig;
-
+            CurrentOptimizelyConfig = new OptimizelyConfigService(CurrentProjectConfig).GetOptimizelyConfig();
```

Comment:
What if GetOptimizelyConfig or OptimizelyConfigService throws exception? Please add it in try catch and move in separate private method. Also log for exception and reinitializing OptimizelyConfigService